neuralmonkey/runners/representation_runner.py

```diff
@@ -0,0 +1,61 @@
+"""A runner that prints out the input representation from an encoder."""
+
+# pylint: disable=unused-import
+from typing import Dict, List, Optional
+# pylint: enable=unused-import
+
+import tensorflow as tf
+
+from neuralmonkey.model.model_part import ModelPart
+from neuralmonkey.runners.base_runner import (BaseRunner, Executable,
+                                              ExecutionResult, NextExecute)
+
+
+class RepresentationExecutable(Executable):
+    def __init__(self, prev_coders: List[ModelPart],
+                 encoded: tf.Tensor) -> None:
+        self._prev_coders = prev_coders
+        self._encoded = encoded
+
+        self.result = None  # type: Optional[ExecutionResult]
+
+    def next_to_execute(self) -> NextExecute:
+        return self._prev_coders, {"encoded": self._encoded}, {}
+
+    def collect_results(self, results: List[Dict]) -> None:
+        if len(results) > 1:
```

Comment:
Nešlo by to vyřešit po šalamounsku a dát tomu runneru parametr "session_index", kterej by byl default nula a kterej by ovládal, ze který sessiony se to vezme?